### DISCUSSION TITLE
Translation: What does a & before or within words mean?

### DISCUSSION THREAD

**Discussion by n1smithy** [Ylianst/MeshCentral#6244] [Q&A]
Hi, people!

I'm trying to improve the German translation. While inspecting the resident translations I found some interesting english lines. Please have a look on the following screenshot:
![Bildschirmfoto vom 2024-07-09 21-25-17](https://github.com/Ylianst/MeshCentral/assets/140073251/27cc4605-2d5a-45a8-91e5-846a5e0d1470)

It's not "Open Site..." but "O&pen Site...". What does the "&" mean? Sometimes it's in front of a specific word or at the beginning of a line. But I cannot see what this means. I know this Symbol as the beginning of a special-character-code within HTML. Are those just mistakes or what's their meaning?

Thanks a lot!

**Comment by si458** [comment]
To me, this looks like a spelling mistake or a completely unused string

One way to verify if the string is used anywhere, is to tick the box saying `Show Location`

This will then add a line saying Locations, and under it show the files and locations of the string

If it shows `No Locations` then the string can't be found anywhere in any of the handlebar files, so u can ignore translating this value

**Comment by n1smithy** [comment]
Yes, you're right. There were some of those strings which are not used. But as there are so many of them I thought there must be a special meaning. OK, then I can leave them untouched. :-)
Thank you!

**Comment by silversword411** [comment]
> What does the "&" mean?

Old school windows dev trick. Makes the `p` be underlined, making it alt-shortcut compatible.

You're making me feel old for knowing this....
